en/gps_compass/gps_cuav_neo_3.md

```diff
@@ -0,0 +1,61 @@
+| Compass                                           | IST8310                                                                        |
+| GNSS receiver                                     | UBLOX NEO M9N                                                                  |
+| RGB drive                                         | NC5623C                                                                        |
+| Buzzer                                            | Passive buzzer                                                                 |
+| Safety switch                                     | Physical button                                                                |
+| GNSS                                              | Beidou, Galileo, GLONASS, GPS                                                  |
+| GNSS Augmentation System                          | SBAS:WAAS,EGNOS,MSAS<br>QZSS:L1s(SAIF)<br>other：RTCM3.3                       |
+| Number of concurrent GNSS                         | 4                                                                              |
+| Frequency band                                    | GPS:L1C/A<br>GLONASS:L10F<br>Beidou:B1I<br>Galileo:E1B/C                       |
+| Horizontal accuracy                               | 2.0M                                                                           |
+| Speed accuracy                                    | 0.05M/S                                                                        |
+| Nav. update rate                                  | 25Hz(Max)                                                                      |
+| Acquisition                                       | Cold start：24S<br>Hot start：2S<br>Aided start:2s                             |
+| Number of satellites(MAX)                         | 32+                                                                            |
+| Sensitivity                                       | Tracking and nav-167dBm<br>Cold start Hot star-148dBm<br>Reacquisition -160dBm |
+| Protocol                                          | UART+IO+I2C                                                                    |
+| Port TYpe                                         | GHR-10V-S                                                                      |
+| Supported flight controller                       | CUAV series,<br>Pixahwk series                                                 |
+| Wave filtering                                    | SAW+LNA+SAW                                                                    |
+| Anti-Electromagnetic/radio frequency interference | EMI+RFI                                                                        |
+| Upgrade firmware                                  | support                                                                        |
+| Support Flight control type                       | Ardupilot/PX4\*(PX4 is still working hard to adapt to the clock)               |

```

Comment:
What does this mean? "Support Flight control type " in the datasheet is something like:

- CUAV series flight control
- Pixhawk series (buzzer does not work)

Either way, what does this mean to users?

Reply:
Please help me remove (buzzer does not work), the previous CUAV GPS internal buzzer is not compatible with pixhawk, but starting from neo 3, it is perfectly compatible with pixhawk standard.

Reply:
Done!